.circleci/config.yml

```diff
@@ -197,3 +190,19 @@ workflows:
           requires:
             - test
             - coverage
+            
+  canary:
+    when:
+      equal: [ canary, << pipeline.parameters.GHA_Action >> ]
+    jobs:
+      - setup
+
+      - build:
+          requires:
+            - setup
+
+      - release:
+          context:
+            - Publish
+          requires:
+            - build
```

Comment:
Should `canary` builds require passing tests? The main rationale for not putting them was speed to publishing, but this actually removes the `test` CircleCI checks from the PR, so adding testing to the canary workflow would ensure that the test status check is always up-to-date.

Reply:
I do see that perspective but I do think we would need some way to make sure that tests are ran at some point before merging.

Reply:
we could run the tests after the canary publish, so the publish is fast but you still get the feedback?